[Feature Request] Last.fm Scrobbler Integration.

Hey, i just wanna say that i love these extras plugins and tools for Youtube Music and i was wondering if you could also add Last.fm Scrobbler Integration in the settings, I would defiantly love to use this for streaming and it would really help me out if this function was added. Again, thank you so much for making this and i hope i can use this to stream music in the future.

Yeah, it's missing. I tried deleting the file hoping it would regenerate and also adding this section, but it doesn't work. It pops up a different error:

                last-fm:true,
                    ^

SyntaxError: Unexpected token '-'
    at wrapSafe (internal/modules/cjs/loader.js:1060:16)
    at Module._compile (internal/modules/cjs/loader.js:1108:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1173:10)
    at Module.load (internal/modules/cjs/loader.js:992:32)
    at Module._load (internal/modules/cjs/loader.js:885:14)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12738)
    at Module.require (internal/modules/cjs/loader.js:1032:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\mathe\Desktop\youtube-music\config\index.js:1:23)
    at Module._compile (internal/modules/cjs/loader.js:1152:30)

and also:

yarn run v1.22.10
$ electron .

App threw an error during load
Error: Cannot find module './defaults'
Require stack:
- C:\Users\mathe\Desktop\youtube-music\config\index.js
- C:\Users\mathe\Desktop\youtube-music\index.js
- C:\Users\mathe\Desktop\youtube-music\node_modules\electron\dist\resources\default_app.asar\main.js
-
    at Module._resolveFilename (internal/modules/cjs/loader.js:972:15)
    at Function.n._resolveFilename (electron/js2c/browser_init.js:249:921)
    at Module._load (internal/modules/cjs/loader.js:848:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12738)
    at Module.require (internal/modules/cjs/loader.js:1032:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\mathe\Desktop\youtube-music\config\index.js:1:23)
    at Module._compile (internal/modules/cjs/loader.js:1152:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1173:10)
    at Module.load (internal/modules/cjs/loader.js:992:32)

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
 	"scripts": {
 		"test": "jest",
 		"start": "electron .",
-        "start:debug": "ELECTRON_ENABLE_LOGGING=1 electron .",
+		"start:debug": "ELECTRON_ENABLE_LOGGING=1 electron .",
 		"icon": "rimraf assets/generated && electron-icon-maker --input=assets/youtube-music.png --output=assets/generated",
 		"generate:package": "node utils/generate-package-json.js",
 		"postinstall": "yarn run icon && yarn run plugins",
@@ -82,7 +82,6 @@
 		"md5": "^2.3.0",
 		"node-fetch": "^2.6.1",
 		"node-notifier": "^9.0.1",
-		"open": "^8.0.3",
 		"ytdl-core": "^4.5.0",
 		"ytpl": "^2.1.1"
 	},

--- a/plugins/last-fm/back.js
+++ b/plugins/last-fm/back.js
@@ -1,6 +1,6 @@
 const fetch = require('node-fetch');
-const { shell } = require('electron')
 const md5 = require('md5');
+const { shell } = require('electron');
 const { setOptions } = require('../../config/plugins');
 const getSongInfo = require('../../providers/song-info');
 const defaultConfig = require('../../config/defaults');

--- a/plugins/last-fm/back.js
+++ b/plugins/last-fm/back.js
@@ -1,6 +1,6 @@
 const fetch = require('node-fetch');
+const { shell } = require('electron')
 const md5 = require('md5');
-const open = require("open");
 const { setOptions } = require('../../config/plugins');
 const getSongInfo = require('../../providers/song-info');
 const defaultConfig = require('../../config/defaults');
@@ -58,7 +58,7 @@ const authenticate = async config => {
 	// asks the user for authentication
 	config.token = await createToken(config);
 	setOptions('last-fm', config);
-	open(`https://www.last.fm/api/auth/?api_key=${config.api_key}&token=${config.token}`);
+	shell.openExternal(`https://www.last.fm/api/auth/?api_key=${config.api_key}&token=${config.token}`);
 	return config;
 }
 

